.github/ISSUE_TEMPLATE/server-migration.md

```diff
@@ -60,44 +63,46 @@ Make sure to clear cache so that instance settings are applied:
 
 ## 3. Migration
 ### preparation
-- [ ] **new server**: `bin/rake db:reset -e production` (important: make sure you're on the new server!)
-- [ ] `deploy.yml -l x_prod2 -e "git_version=vX.Y.Z"` matching version with current prod
+- [ ] Reset database on new server, to avoid any migration issues due to being out of sync
+  `bin/rake db:reset` (You will need to confirm. Make sure you're on the new server!)
+- [ ] Update ansible_host IP in `inventory/hosts` and ensure provision works (this will update host in `.env.production`).
+    `ansible-playbook playbooks/provision.yml -l x_prod`
+- [ ] `ansible-playbook playbooks/deploy.yml -l x_prod -e "git_version=vX.Y.Z"` matching version with current prod
 - [ ] old server: make a tiny data change to verify later (eg add `.` in meta description `/admin/general_settings/edit`)
 
 ### switchover: old server
-- [ ] 🚧 `maintenance_mode.yml`
+- [ ] 🚧 `ansible-playbook playbooks/maintenance_mode.yml -l x_prod`
 - [ ] `sudo systemctl stop sidekiq redis-jobs puma`
```

Comment:
Should we add `disable` as well? Just in case the server gets restarted accidentally. It means that rolling back is also one step more (`enable`) but it makes the intent of retiring the server at this stage clearer. And it could prevent us from accidentally running a start or restart command on the old server.